src/spikeinterface/core/baserecording.py

```diff
@@ -746,6 +746,30 @@ def _select_segments(self, segment_indices):
 
         return SelectSegmentRecording(self, segment_indices=segment_indices)
 
+    def get_channel_locations(
+        self,
+        channel_ids: list | np.ndarray | tuple | None = None,
+        axes: "xy" | "yz" | "xz" = "xy",
+    ) -> np.ndarray:
+        """
+        Get the physical locations of specified channels.
```

Comment:
I think this is likely improvement enough for merging, but since contact vs channel is still a point of confusion for many users we may want to workshop better ways to talk about channel_locations for public functions.